Applications/YOLO/jni/main.cpp

```diff
@@ -254,6 +253,17 @@ int main(int argc, char *argv[]) {
     model->summarize(std::cout, ML_TRAIN_SUMMARY_MODEL);
 
     model->train();
+
+    // // simple inference example
+    // model->load("./best_model.bin");
```

Comment:
I think we need to add load function to load weight bin file before train as well

Reply:
I think there should be a pre-trained weights bin file to load before train.